bridges/modules/grandpa/src/mock.rs

```diff
@@ -59,6 +59,7 @@ impl frame_system::Config for TestRuntime {
 	type RuntimeOrigin = RuntimeOrigin;
 	type Nonce = u64;
 	type RuntimeCall = RuntimeCall;
+	type RuntimeTask = RuntimeTask;
```

Comment:
@kianenigma is it OK that right now it requires this everywhere?

Reply:
Yes that's understandable.

Reply:
I think you should not spend too time on this for now. Hopefully https://github.com/paritytech/polkadot-sdk/issues/171 is done before you are done.